src/routes/paths.ts

```diff
@@ -30,7 +30,7 @@ export const EXTERNAL_URL_MEDIUM_ARTICLE =
   'https://medium.com/iearn/decentralized-payroll-management-for-daos-b2252160c543';
 // TODO: Change this to something more specific to feedback.
 export const EXTERNAL_URL_FEEDBACK =
```

Comment:
```suggestion
export const EXTERNAL_URL_WHY_COORDINAPE_IN_CIRCLE =
```

Reply:
There is some other places this would need to change, but I feel like the previous name (even tho out of scope of this PR) is kinda weird